docs/_articles/en/apex/writing.md

```diff
@@ -25,7 +25,71 @@ Out of the box snippets for Salesforce development are available in these reposi
 - [Apex Code Snippets](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex/snippets/apex.json)
 - [Code Snippets for LWC development](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-lwc/snippets/lwc.json) 
 
-If you want to define your own code snippets, read [Snippets in Visual Studio Code](https://code.visualstudio.com/docs/editor/userdefinedsnippets).
```

Comment:
Nice! I like adding more specifics on custom snippets here, rather making users wander off to find this information.